src/dmd/access.d

```diff
@@ -612,7 +612,23 @@ public Dsymbol mostVisibleOverload(Dsymbol s)
         else
             break;
 
-        if (next && mostVisible.prot().isMoreRestrictiveThan(next.prot()))
+        /**
+        * Return the "effective" protection attribute of a symbol when accessed in a module.
+        * The effective protection attribute is the same as the regular protection attribute,
+        * except package() is "private" if the module is outside the package;
+        * otherwise, "public".
+        */
+        Prot protectionInModule(Dsymbol d, Module mod = null)
```

Comment:
Looks like this function can be `static`.

Reply:
I still prefer "From" over "In".

Reply:
fixed?